lib/SILOptimizer/Mandatory/TFPartition.cpp

```diff
@@ -2014,7 +2012,8 @@ bool TFFunctionPartition::markFunction(bool &hasTensorOps) {
   };
 
   // Split all critical edges for block marking.
-  splitAllCondBrCriticalEdgesWithNonTrivialArgs(hostFn, &DI, &LI);
+  // FIXME: Remove this when deabstraction is fully implemented.
+  splitAllCondBrCriticalEdgesWithNonTrivialArgs(hostFn, &DI, /*LI*/ nullptr);
```

Comment:
can we also remove `#include "swift/SILOptimizer/Analysis/LoopAnalysis.h"` now?

Reply:
Good catch. Yes!